pages/hugo.yml

```
@@ -30,7 +30,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      HUGO_VERSION: 0.99.0
+      HUGO_VERSION: 0.102.1
```

There have already been 2 additional patch releases since this PR was created. 😅 

```suggestion
      HUGO_VERSION: 0.102.3
```